### PR TITLE
Fix backend compilation + adding fallback for docker container

### PR DESCRIPTION
**Pull request by vnepogodin**
Fixes https://github.com/AppFlowy-IO/appflowy/issues/91

### CODE REVIEW

File: backend/src/middleware/auth_middleware.rs [hunk 1]
```diff
@@ -87,7 +87,7 @@ where
             let fut = self.service.call(req);
             return Box::pin(async move {
                 let res = fut.await?;
-                Ok(res.map_body(|_, body| AnyBody::from_message(body)))
+                Ok(res.map_body(|_, body| AnyBody::new_boxed(body)))
```
Comment (appflowy) [inline backend/src/middleware/auth_middleware.rs:90]: It's weird, I can't find the definition of new_boxed.

Comment (vnepogodin) [reply backend/src/middleware/auth_middleware.rs:90]: https://docs.rs/actix-http/3.0.0-beta.13/src/actix_http/body/body.rs.html#51
https://docs.rs/actix-http/3.0.0-beta.13/src/actix_http/body/body.rs.html#236

Comment (appflowy) [reply backend/src/middleware/auth_middleware.rs:90]: Got it. I think the version of actix-web should be updated in the cargo.toml. 

Current:
actix-web = "4.0.0-beta.11"
actix-http = "3.0.0-beta.12"

It seems like the beta version will be updated automatically if the Cargo.lock does not exist.

**Comment by appflowy** [comment]
I think I should add the Cargo.lock to git, as described in https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

> If you’re building a non-end product, such as a rust library that other rust packages will depend on, put Cargo.lock in your .gitignore. If you’re building an end product, which are executable like command-line tool or an application, or a system library with crate-type of staticlib or cdylib, check Cargo.lock into git.

**Comment by vnepogodin** [comment]
Well build is fixed. So I'll revert commit

**Pull request closed without merging**